runtime/compiler/trj9/runtime/codertinit.cpp

```diff
@@ -511,6 +512,7 @@ void codert_init_helpers_and_targets(J9JITConfig * jitConfig, char isSMP)
 #endif
    jitConfig->jitDiscardPendingCompilationsOfNatives = jitDiscardPendingCompilationsOfNatives;
    jitConfig->jitMethodBreakpointed = jitMethodBreakpointed;
+   jitConfig->jitIllegalFinalFieldModification = jitIllegalFinalFieldModification;
```

Comment:
This is not the file to fill in the function pointer - it should be somwhere in the JIT.  Ask Andrew.

Reply:
this seems to be the only place that jitConfig function pointers are set? it is not part of JIT?

Reply:
Look in rossa.cpp for:

   // set up the entryPoints for compiling and for support of the
   // java.lang.Compiler class.

Reply:
And I suppose you could move the dummy function in there for simplicity - the JIT implementors can move it later if need be.  Make the function declaration static.